vaadinfrontend/src/main/java/life/qbic/datamanager/views/project/overview/ProjectOverviewLayout.java

```diff
@@ -25,42 +29,45 @@
 @PageTitle("Project Overview")
 @Route(value = "projects", layout = MainLayout.class)
 @PermitAll
-public class ProjectOverviewLayout extends VerticalLayout {
+public class ProjectOverviewLayout extends Composite<CardLayout> {
 
-    Button create;
-    Grid<ProjectPreview> projectGrid;
-    OfferSearchDialog searchDialog;
+    @Serial
+    private static final long serialVersionUID = 5435551053955979169L;
+    Button create = new Button("Create");
+    Grid<ProjectPreview> projectGrid = new Grid<>(ProjectPreview.class, false);
+    OfferSearchDialog searchDialog = new OfferSearchDialog();
 
-    TextField projectSearchField;
+    TextField projectSearchField = new TextField();
 
-    CreationModeDialog selectCreationModeDialog;
+    private CardLayout cardLayout = new CardLayout();
```

Comment:
why do you have a `CardLayout` here? I thought you moved to the Composite<CardLayout>. In this case, you can use `getContent()` to get the card layout.